### ISSUE TITLE
imagine hacking an io game which even the owner doesnt care about. 

### ISSUE THREAD

**Issue by codersalty** [IceHacks/SurvivCheatInjector#657]
imagine hacking an io game which even the owner doesnt care about. like i mean whats the point is it to prove that you are very skilled devs? well then make something useful your not going to get into mit or google by making a hack for an io game with no anticheat. is it for fun? idk what your meaning of "fun" is but if ruining peoples lives are your idea of "fun" well then you guys have a lot to learn about life

**Comment by divinelemon** [comment]
To address your point:

> ...by making a hack for an io game with no anticheat.

The devs at Surviv.io have released anticheats before (and maybe are still? see #658). 

> is it for fun?

It _is_ pretty fun to pretend you're a pro player. I use the autoaim and get a lot of prestige because people think that I can just play the game (the autoaim is way too blatant, a normal player doesn't twitch like the autoaim does, but autoaim looks real).

> ... but if ruining peoples lives are your idea of "fun" ...

I mean come on. This is an IO game. There are hackers in competitive games that have actual $ prizes, and even those hackers are accepted as a part of the game. It's the same way in Surviv.io. I used to actually play (I was a pro) and I would have fun killing hackers (at the time the melee wasn't super powerful/op).

Basically, nobody really cares. The devs are silently pushing anticheats. The guys at IceHacks make more cheats. It's just the way to world works.

**Comment by IceHacks** [comment]
imagine caring about someone cheating on a .io-game that the owner doesn't even care about.

**Comment by codersalty** [comment]
imagine replying to someone mad about someone cheating on a .io-game that the owner doesn't even care about.
